plugins/chat/assets/javascripts/discourse/components/chat-composer-message-details.hbs

```diff
@@ -1,6 +1,6 @@
-<div class="chat-composer-message-details">
+<div class="chat-composer-message-details" data-id={{@message.id}}>
   <div class="chat-reply">
-    {{d-icon @icon}}
+    {{d-icon (if @message.editing "pencil-alt" "reply")}}
```

Comment:
I move all this logic inside the component instead of having it from the outside